neptune/new/exceptions.py

```diff
@@ -45,6 +45,29 @@ class MetadataInconsistency(NeptuneException):
     pass
 
 
+class MissingAttributeException(NeptuneException, AttributeError, KeyError):
+    """Raised when get-like action is called on `Handler`, instead of on `Attribute`."""
+
+    def __init__(self, field_path):
+        message = """
+{h1}
+----MissingAttributeException---------------------------------------------------
```

Comment:
Let's change this to `MissingFieldException` or `KeyErrorException` or sth like this. We don't promote the word Attribute externally. @aniezurawski could advise (I remember him having some ideas).